~/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-api-gateway ~/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-api-gateway
----------------------------------------------------
Running Python linter against following directories:
gateway
tests
tools
----------------------------------------------------

created virtual environment CPython3.6.10.final.0-64 in 159ms
  creator CPython3Posix(dest=/home/msawood/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-api-gateway/venv, clear=False, global=False)
  seeder FromAppData(download=False, pip=latest, setuptools=latest, wheel=latest, via=copy, app_data_dir=/home/msawood/.local/share/virtualenv/seed-app-data/v1.0.1)
  activators PythonActivator,FishActivator,XonshActivator,CShellActivator,PowerShellActivator,BashActivator
Requirement already satisfied: pycodestyle in ./venv/lib/python3.6/site-packages (2.5.0)
gateway/__init__.py
    Pass
gateway/errors.py
    Pass
gateway/auth.py
    Pass
gateway/defaults.py
    Pass
gateway/api_v1.py
    Pass
tests/test_auth.py
    Pass
tests/test_defaults.py
    Pass
tests/test_errors.py
    Pass
tests/test_api_v1.py
    Pass
tools/check_python_version.py
    Pass
All checks passed for 10 source files
~/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-api-gateway
